docs/articles/multi-attribute-semantic-search.md

```diff
@@ -0,0 +1,459 @@
+    .limit(LIMIT)
+)
+```
+
+Finally, start executor and upload the data:
+
+```python
+monster_parser = DataFrameParser(monster, mapping={monster.id: "name"})
+
+source: InMemorySource = InMemorySource(monster, parser=monster_parser)
+executor = InMemoryExecutor(sources=[source], indices=[monster_index])
+app = executor.run()
+
+source.put([df])
+```
+
+Let's run the same query as before:
+
+```python
+query = {
+    'look': 'glowing',
+    'habitat': 'dark places',
+    'behavior': 'light manipulation'
+}
+```
+
+| id               |    score | look                                                           | habitat                                             | behavior                                                      |
+|:-----------------|---------:|:---------------------------------------------------------------|:----------------------------------------------------|:--------------------------------------------------------------|
+| Whispering Shade | 0.376738 | Shadowy, amorphous being with glowing eyes                     | Dark forests and abandoned buildings                | Feeds on fear and whispers unsettling truths                  |
+| Luminoth         | 0.340084 | Moth-like creature with glowing wings and antenna              | Dense forests and jungles with bioluminescent flora | Emits soothing light patterns to communicate and attract prey |
+| Living Graffiti  | 0.330587 | Two-dimensional, colorful creature that inhabits flat surfaces | Urban areas, particularly walls and billboards      | Shapeshifts to blend with surroundings and absorbs pigments   |
+
+As you can see, the results contain monsters that satisfy all criteria at once.
+
+Let's try another query.
+
+```python
+query = {
+    'look': 'shapeshifting',
+    'habitat': 'varied landscapes',
+    'behavior': 'illusion creation'
+}
+```
+
+| id               |    score | look                                               | habitat                                   | behavior                                             |
+|:-----------------|---------:|:---------------------------------------------------|:------------------------------------------|:-----------------------------------------------------|
+| Mist Phantom     | 0.489574 | Ethereal, fog-like humanoid with shifting features | Swamps, moors, and foggy coastlines       | Lures travelers astray with illusions and whispers   |
+| Zephyr Dancer    | 0.342075 | Graceful avian creature with iridescent feathers   | High mountain peaks and wind-swept plains | Creates mesmerizing aerial displays to attract mates |
+| Whispering Shade | 0.337434 | Shadowy, amorphous being with glowing eyes         | Dark forests and abandoned buildings      | Feeds on fear and whispers unsettling truths         |
+
+Great!
+Maybe we would like to find monsters from the dataset similar to the existing one?
+Let's try it with the "Harmonic Coral" (very last row in our dataset).
+
+```python
+query = {
```

Comment:
This seems like a with_vector query? I think it is worth signalling that this is run agains a different query that was defined.